public/views/MainView.js

```diff
@@ -0,0 +1,23 @@
+var Backbone = require('backbone');
+
+var mainRouter = require('../routers/mainRouter');
+var mainController = require('../controllers/mainController');
+
+module.exports = Backbone.View.extend({
+  events: {
+    //add something
+  },
+  initialize: function(e) {
+    console.log('main view working');
+    this.mainRouter = new mainRouter({ mainController: mainController });
+    // this.mainRouter.on
+    // pushState: true,
+    // something is wrong with hash redirects
+    // need prevent default: http://stackoverflow.com/questions/28323836/backbone-js-pushstate-true
+    // help here? http://slides.com/kinakuta/deck-17?token=vY03XdtR#/8
+    // may need vent
+
+    // Backbone.history.start({ hashChange: false, pushState: true });
+    Backbone.history.start();
```

Comment:
You had the right idea:

`Backbone.history.start({ pushState: true });`

The part that's missing is you don't have event handling for things that would result in client-side routing, like clicking on the link that has an href of `/planting` You need a view that would handle clicking on that link that would then trigger an event that could be handled by your controller, which could then do something, or the view handling the click itself would do something. That all depends on how you want to structure things.